src/test/daml/Daml/Finance/Instrument/Swap/Test/Fpml.daml

```diff
@@ -445,3 +445,175 @@ runStubRateInterpolation = script do
   swapInstrumentAfterFirstPayment <- lifecycleAndVerifySwapPaymentEffects [publicParty] (addDays firstRegularPeriodDate 1) swapInstrument settlers issuer investor observers custodian [observableCid, observableOneMonthCid] expectedConsumedQuantities expectedProducedQuantities
 
   pure ()
+
+-- Test sample trade with two stub periods (intial and final) in the same trade.
+runDualStubSampleTrade : Script ()
+runDualStubSampleTrade = script do
+  [custodian, issuer, investor, calendarDataProvider, publicParty] <-
+    createParties ["Custodian", "Issuer", "Investor", "Calendar Data Provider", "PublicParty"]
+
+  -- Account and holding factory
+  let fp = [("FactoryProvider", singleton publicParty)]
```

Comment:
```suggestion
  let fp = [("FactoryProvider", singleton publicParty)]
```

This line is redundant, as it is the same as `observers` defined line 460, pls remove

Reply:
Thanks, I removed it from the other tests as well.